opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py

```diff
@@ -1347,25 +1361,13 @@ class ExplicitBucketHistogramAggregation(Aggregation):
 
     def __init__(
         self,
-        boundaries: Sequence[float] = (
-            0.0,
-            5.0,
-            10.0,
-            25.0,
-            50.0,
-            75.0,
-            100.0,
-            250.0,
-            500.0,
-            750.0,
-            1000.0,
-            2500.0,
-            5000.0,
-            7500.0,
-            10000.0,
-        ),
+        boundaries: Optional[Sequence[float]] = None,
```

Comment:
Any reason to not leave the default as `_DEFAULT_EXPLICIT_BUCKET_HISTOGRAM_AGGREGATION_BOUNDARIES`? I'm wondering because this allows explicitly passing `None` now

Reply:
@aabmass Yeah, now you can pass None boundaries but then the defaults boundaries are used. Also this is more pythonic I guess?

Reply:
What's the reason user would pass None instead of allowing the default?

Reply:
There's no reason the user would pass None, I expected to just call it without passing the parameter as currently doing. But no big deal in keeping thing as is.

Reply:
Well, it looks like we have someone passing None: 

```
    def __init__(self, boundaries: Sequence[float], **kwargs) -> None:
>       super().__init__(len(boundaries) + 1, **kwargs)
E       TypeError: object of type 'NoneType' has no len()

opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/exemplar/exemplar_reservoir.py:296: TypeError
```

and

```
>       self._boundaries = tuple(boundaries)
E       TypeError: 'NoneType' object is not iterable

opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py:488: TypeError
```

Reply:
I'm fine with whatever you want to do here

Reply:
The errors come from here I guess, without the `default=None` I would add to compose the **kwargs by hand and optionally add the boundaries arg:
https://github.com/open-telemetry/opentelemetry-python/pull/4361/files#diff-f47777d9ac6e010c971c2252915bcbc8ec95dd61d24428c88d52b3f08a31f745R1286-R1290